src/main/java/Duke.java

```diff
@@ -1,10 +1,139 @@
+    public static final int DEADLINE_SIZE = 9;
+    public static final Task[] tasks      = new Task[100];
+    public static final String LINE       = "--------------------------------------------------------------------------------";
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
-    }
-}
+
+        // Prints logo and welcome message
+        printWelcome();
+
+        int itemIndex = 0;
+        boolean saidBye = false;
+
+        Scanner in = new Scanner(System.in);
+
+        // Cond. is true when input "bye" is given, terminates run
+        while (!saidBye) {
+
+            // Takes in user commands to interact with bot
+            String command = in.nextLine();
+
+            if (command.equals("bye")) {
+                saidBye = true;
+                printBye();
+            } else if (command.equals("list")) {
+                printList(tasks, itemIndex);
+            } else if (command.contains("done")) {
+                doneMethod(tasks, itemIndex, command);
+            } else if (command.equals("echo")) {
+                echoMode();
+            } else {
+                // Create tasks for class Todo
+                if (command.contains("todo")) {
+                    // Reads substring containing the task description only
+                    tasks[itemIndex] = new Todo(command.substring(TODO_SIZE));
+                }
+                // Create tasks for class Deadline
+                else {
+                    final String dateOrTime = command.substring(command.indexOf("/") + DASH_INDX);
+                    if (command.contains("deadline")) {
+                        // Reads two substrings as param: 1. The task description after keyword "deadline"
+                        //                                2. The actual deadline after "/by " till end of string
+                        tasks[itemIndex] = new Deadline((command.substring(DEADLINE_SIZE, command.indexOf("/"))),
+                                dateOrTime);
+                    }
+                    // Create tasks for class Event
+                    else if (command.contains("event")) {
+                        // Same substring read as Deadline
+                        tasks[itemIndex] = new Event((command.substring(EVENT_SIZE, command.indexOf("/"))),
+                                dateOrTime);
+                    }
+                    // If no keywords are used, stores new tasks into Task array
+                    else {
+                        tasks[itemIndex] = new Task(command);
+                    }
+                }
+                itemIndex++;
+                System.out.println("Will do sir, I've added: " + System.lineSeparator() + "  "
```

Comment:
Any reason why you decided to use a mix of `System.lineSeparator()` and `"\n"`? I have noticed this possible inconsistency in several other places as well.